src/Jackett.Common/Definitions/monova.yml

```
@@ -49,9 +49,9 @@ search:
     # https://monova.to/search?term=vikings+s05e05
     # https://monova.to/video
     - path: "{{if .Keywords}}search?term={{.Keywords}}{{else}}video{{end}}"
-    - path: "{{if .Keywords}}search?term={{.Keywords}}{{else}}video{{end}}&page=2"
-    - path: "{{if .Keywords}}search?term={{.Keywords}}{{else}}video{{end}}&page=3"
-    - path: "{{if .Keywords}}search?term={{.Keywords}}{{else}}video{{end}}&page=4"
+    - path: "{{if .Keywords}}search?term={{.Keywords}}{{else}}video?{{end}}&page=2"
```

```suggestion
    - path: "{{ if .Keywords }}search?term={{ .Keywords }}&{{ else }}video?{{end}}page=2"
```
normally an ampersand does not follow a question mark.

---

it worked, but it definitely didn't look right